recipe/run_test.sh

```diff
@@ -56,7 +56,6 @@ unittest() {
 }
 
 test "${CONDA_BUILD:-}" = 1 && cd ../test_files || cd $(dirname $0)/../src
-msg Running in $PWD
```

Comment:
While working on this PR, I iterated on the tests a lot and was annoyed by this not-very-useful message, so removed it.